gammapy/data/tests/test_spectral_cube.py

```diff
@@ -162,7 +161,7 @@ def test_integral_flux_image(self):
     def test_solid_angle_image(self):
         actual = self.spectral_cube.solid_angle_image[10][30]
         expected = Quantity(self.spectral_cube.wcs.wcs.cdelt[:-1].prod(), 'deg2')
-        assert_quantity(actual, expected.to('sr'), rtol=1e-4)
+        assert_quantity_allclose(actual, expected.to('sr'), rtol=1e-4)
```

Comment:
The `.to('sr')` part is superfluous and can be removed, no?
I.e. the same test with the same precision is performed if it's there or not?